tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -88,6 +88,13 @@ define(KWD_PIPE_SCH_DEADLINE_US, 20000)
 # include the generic dmic with kwd
 include(`platform/intel/intel-generic-dmic-kwd.m4')
```

Comment:
Hmm, what's "FB#" in the description. Full Bandwidth? I've not seen that used, so maybe replace with "support Bluetooth sample rates on SSP port 2".

Reply:
Ack, the different bandwidth modes (NB, WB, SWB, FB) are not commonly recognized.